### PR TITLE
Update ported rules

### PR DESCRIPTION
**Pull request by reflectronic**
Updates the list of ported rules.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@reflectronic : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/code-quality/fxcop-rule-port-status.md [hunk 1]
```diff
@@ -95,6 +102,7 @@ CA1509 | Invalid entry in code metrics rule specification file
 [CA1725](ca1725.md) | Parameter names should match base declaration
 [CA1801](ca1801.md) | Review unused parameters
 [CA1802](ca1802.md) | Use literals where appropriate
+[CA1804](ca1804.md) | Remove unused locals
```
Comment (mavasani) [inline docs/code-quality/fxcop-rule-port-status.md:105]: `CA1804` was ported, but then removed and deprecated in favor of another rule in IDE CodeStyle NuGet package. See https://github.com/dotnet/roslyn-analyzers/issues/2109 for details.

File: docs/code-quality/fxcop-rule-port-status.md [hunk 2]
```diff
@@ -25,7 +25,9 @@ Rule ID | Title
 --------|---------
 [CA1000](ca1000.md) | Do not declare static members on generic types
 [CA1001](ca1001.md) | Types that own disposable fields should be disposable
+[CA1002](ca1002.md) | Do not expose generic lists
```
Comment (mavasani) [inline docs/code-quality/fxcop-rule-port-status.md:28]: Tagging @Evangelink to verify the list as we he was the one who ported majority, if not all, of these rules.

File: docs/code-quality/fxcop-rule-port-status.md (hunk 1 above)
Comment (reflectronic) [reply docs/code-quality/fxcop-rule-port-status.md:105]: Alright, I'll move it to the bottom of the page then.

**Comment by PRMerger6** [comment]
@reflectronic : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
@Evangelink Can you take a quick look at this? Thanks.

**Comment by Evangelink** [comment]
@Mikejo5000 @mavasani Looking (only) at the merged PR I have opened, I can see that `CA1021` is missing from the list: https://github.com/dotnet/roslyn-analyzers/pull/3176

**Comment by reflectronic** [comment]
CA1021, for whatever reason, was already both in the "ported" and "not ported" list. That is why I did not add to the other list when removing it.

**Comment by Evangelink** [comment]
@reflectronic My bad! 

@Mikejo5000 It's all good for me.

**Comment by ktoliver** [comment]
Hi @Mikejo5000 -- Is this pull request still in progress? It looks like master has been updated to include the list items added here, but master uses a different link:

image

Thanks.

**Comment by mavasani** [comment]
Yep, the ported rules have been updated and moved to dotnet/docs repo. We should close this PR.

**Comment by ktoliver** [comment]
> Yep, the ported rules have been updated and moved to dotnet/docs repo. We should close this PR.

Got it, thanks. #please-close

**Pull request closed without merging**